src/views/wallet/UserAssetsScreen.tsx

```diff
@@ -1,7 +1,51 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { Row, Col, Table } from 'antd'
+import DynamicCoin from '../../components/shared/CoinIcons/DynamicCoin'
 
-export default function UserAssetsScreen () : JSX.Element {
+type UserAssetType = {
+  _id: string;
```

Comment:
^ Just curious: Is there any reason to use an underscore here?

Reply:
It's a convention used with JSON/MongoDB in which fields with leading underscores are considered readonly/set-on-insert and related only to internal DB operations rather than that actual data stored in the model. Maybe not relevant in this instance? It was the model I took from the MongoDB I was using previously.